lib/rucio/core/replica.py

```diff
@@ -799,6 +811,37 @@ def _list_replicas_for_files(file_clause, state_clause, files, rse_clause, updat
             {'scope': scope, 'name': name} in files and files.remove({'scope': scope, 'name': name})
 
 
+def get_multi_cache_prefix(cache_site, filename, logger=logging.log):
+
+    # print('Looking up prefix for cache:', cache_site, 'and filename:', filename)
+    x_caches = read_from_cache('CacheSites', expiration_time=60)
+    if not x_caches:
+        # print('reloading xcache sites and ranges.')
+        vp_endpoint = 'https://vps.cern.ch/serverRanges'
+        response = requests.get(vp_endpoint, verify=False)
+        if response:
+            # print('Success! Got XCache ranges.')
+            x_caches = response.json()
+            # print(x_caches)
+            write_to_cache('CacheSites', x_caches)
+        else:
+            logger(logging.ERROR, 'Could not reload from vps.')
```

Comment:
Please make this error more useful. Include this method namely and e. g. the requested http address and the status code.